src/models.py

```diff
@@ -0,0 +1,47 @@
+import uuid
+from sqlalchemy import Boolean, Column, ForeignKey, String
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import relationship
+
+from src.service import Base
+
+
+class Projects(Base):
+    __tablename__ = "projects"
+
+    project_id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    name = Column(String, index=True)
+    description = Column(String, index=True)
```

Comment:
Could you elaborate why we need index for these fields? I dont think you need them
Also explicitly specify if columns are nullable or not, it is better than implicit. Additionally your pydantic related schema for project has nullable description, here it is not